docs/en/integrations/language-clients/python/index.md

```diff
@@ -0,0 +1,316 @@
+---
+sidebar_label: ClickHouse Connect
+sidebar_position: 1
+keywords: [clickhouse, python, client, connect, integrate]
+slug: /en/integrations/python
+description: A suite of Python packages for connecting Python to ClickHouse
+---
+
+# ClickHouse Connect
+
+## Introduction
+ClickHouse Connect is a suite of packages providing interoperability with a wide range of Python applications.
+The three primary components are:
+
+- A low level driver in the package `clickhouse_connect.driver`.  This package includes a basic client that handles 
+all ClickHouse Connect requests to the ClickHouse Server, as well assorted helper classes and utility functions.
+- A limited SQLAlchemy dialect in the package `clickhouse_connect.cc_sqlalchemy`.  This package focuses
+implements query/cursor functionality, and does not generally support SQLAlchemy DDL and ORM operations.
+(SQLAlchemy is targeted toward OLTP databased, and we recommend more specialized tools and frameworks to manage
+the ClickHouse OLAP database.)
+- A Superset EngineSpec in the `clickhouse_connect.cc_superset`.  This package will automatically add a **ClickHouse
+Connect** Superset connector when ClickHouses Connet installed.  This EngineSpec supports all core Superset query
+functionality, but does not currently support certain advanced features such as file upload to a ClickHouse table.
+
+## Requirements and Compatibility
+
+ClickHouse Connect requires Python 3.7 or greater.  It has been tested on Linux (x86 and Aarch64), Windows, and Mac OS
+(x86 and M1), but should run on most Python installations.
+
+ClickHouse Connect uses the ClickHouse HTTP protocol and has been tested against all currently supported ClickHouse
```

Comment:
Shouldn't we list compatible ClickHouse versions?

Reply:
There are definitely some more detailed docs coming, but I don't think they'll end up on separate pages quite yet.